src/Tickets/Emails/Legacy.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace TEC\Tickets\Emails;
+
+use Tribe__Tickets__Tickets as Tickets_Module;
+
+/**
+ * Class Legacy
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Legacy {
+
+	/**
+	 * Send RSVPs/tickets email for an attendee by injecting itself into the legacy Tickets codebase.
```

Comment:
Legacy is the `Tribe__Tickets__Tickets` class, in the codebase that is Legacy, I would like to start shirking that class and eventually have us deprecate it. I know it will take a long time but that is the architecture that I would like for us to aim at.

Reply:
Is the future plan to integrate a lot of what the `Legacy` class is doing into the `Email_Sender` class or somewhere else?

Reply:
Possibly, a lot of the logic there should not be part of the email piece. Determining if we use the RSVP or the Tickets, or Going not Going should actually happen on the Commerce level. Like which email are we sending should not be part of the logic of the Email System. It should be part of the business logic which tends to be part of the Commerce. 

The only problem is currently on the `Tribe__Tickets__Tickets` these domains are merged, which makes it impossible for you to maintain the code without feeling like you are handling a house of cards.